ubuntu_image/helpers.py

```diff
@@ -127,6 +130,39 @@ def snap(model_assertion, root_dir, channel=None, extra_snaps=None):
     run(cmd, stdout=None, stderr=None, env=os.environ)
 
 
+def live_build(root_dir, env):
+    # First, setup the build tools and workspace.
+    config_dir = os.path.join(root_dir, 'auto')
+    shutil.copytree('/usr/share/livecd-rootfs/live-build/auto', config_dir)
```

Comment:
I wonder if we can somehow avoid hard-coding the live-build/auto directory path here. Is there any merit in making this configurable, with the default pointing at the livecd-rootfs one? Will have to think about it. Anyway, would be awesome if we could at least query the system somehow to tell us the auto scripts path from livecd-roofts's dpkg installation.